ui/DesignSystem/Component/Transaction/Center.svelte

```diff
@@ -22,19 +23,21 @@
     }
   };
 
+  $: listIn = { duration: 360 };
+  $: listOut = { duration: 240 };
```

Comment:
This could be just `let listIn` and `let listOut`, right? (In case you decide against inlining).